DragaliaAPI/Features/Login/LoginController.cs

```diff
@@ -15,27 +16,37 @@ namespace DragaliaAPI.Features.Login;
 [Consumes("application/octet-stream")]
 [Produces("application/octet-stream")]
 [ApiController]
+[BypassDailyReset]
```

Comment:
Should this be applied to any other controllers? Anything before you get to /login/index might throw you into an infinite loop, right?

Reply:
DailyResetMiddleware additionally uses the AllowAnonymous attribute for filtering out endpoints that should always be available, meaning that before /tool/auth is called (which refreshes the login time in the session so unblocks any blocked endpoint) all endpoints are accessible